easybuild/easyconfigs/a/ASE/ASE-3.19.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.19.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Tkinter', '%(pyver)s', '-Python-%(pyver)s'),
+    ('matplotlib', '3.0.0', '-Python-%(pyver)s'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# pytest-xdist does not install a separate module, but a plugin to pytest.
+# It therefore cannot be imported during the sanity check.
```

Comment:
what does this comment refer to?

Reply:
Sorry, that was a stale comment.  I removed pytest-xdist again, as it is only the developer version (coming 3.20.0) that uses it.  I forgot to remove the comment.